src/main/java/com/redhat/darcy/salesforce/UrlOutputField.java

```diff
@@ -0,0 +1,82 @@
+/*
+ Copyright 2014 Red Hat, Inc. and/or its affiliates.
+
+ This file is part of darcy-salesforce.
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.redhat.darcy.salesforce;
+
+import static com.redhat.darcy.ui.By.xpath;
+import static com.redhat.darcy.ui.Elements.link;
+
+import com.redhat.darcy.ui.AbstractViewElement;
+import com.redhat.darcy.ui.annotations.Require;
+import com.redhat.darcy.ui.api.Locator;
+import com.redhat.darcy.ui.api.elements.Element;
+import com.redhat.darcy.ui.api.elements.Link;
+
+/**
+ * An HTML link element for a value that corresponds to a URL field on 
+ * a Salesforce object.
+ */
+
+public class UrlOutputField extends AbstractViewElement implements Link {
+    @Require
+    private Element parent = super.parent;
+    
+    private Link nestedLink = link(byInner(xpath("./div/a | ./a")));
```

Comment:
This is kind of an interesting case.  If the field in Salesforce has no value, the anchor tag isn't in the DOM.  Making the link required will throw a time out exception.  So it gets marked NotRequired.  If I also take out the parent element <td>, then it throws a NoRequiredElementsException